.github/workflows/arm64.yml

```diff
@@ -67,6 +67,14 @@ jobs:
           ${TAS_SCRIPTS}/wait-for-alfresco-start.sh "http://localhost:8080/alfresco/" 180
           docker ps
           curl -v --user admin.pipeline@alfresco.com:admin http://localhost:8080/alfresco/api/discovery
+      - name: "Persist environment setup failure logs"
+        id: persist_environment_failure_logs
+        run: |
+          # Get containers state
+          docker ps -a
+          # Get Alfresco repo logs
+          docker ps -aqf "ancestor=quay.io/alfresco/alfresco-pipeline-all-amps-repo:${TEST_GIT_TAG:-latest}" | xargs -L 1 docker logs --tail 3000
```

Comment:
unless for some reason it could become dynamic, right now even a simple:

```
docker logs environment-alfresco-1
```

will be enough.

An alternative approach could be to iterate on all the created containers (which will be only the ones of the current run), dump the logs in a different file for each container, make a tar and then use actions/upload-artifact so it would be available directly in the GHA summary to be downloaded and inspected locally

Reply:
Nice discovery! 👍 Removed image tag

Reply:
Regarding @gionn idea I need to think about it 🤔

Reply:
@gionn I've followed your suggestion and even extended it - introducing new GH action allowing to dump containers logs: https://github.com/Alfresco/alfresco-build-tools/pull/266 . It will allow to use this logic in different workflows in the future.